package.json

```diff
@@ -127,7 +128,7 @@
     "chai-string": "^1.4.0",
     "cheerio": "^1.0.0-rc.10",
     "chokidar-cli": "latest",
-    "cypress": "3.1.5",
+    "cypress": "5.0.0",
```

Comment:
Would we want a `^` for Cypress?

Reply:
We now set a specific Cypress version in [the project Jenkinsfile](https://github.com/alphagov/pay-frontend/pull/2659/commits/256241c678e34af2f5a3ec53d6e861bcc179c2c6#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8) that needs to match the version in the package-lock so the version is pinned.